src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexRunner.cs

```diff
@@ -345,9 +346,10 @@ private Match TidyMatch(bool quick)
         /// </summary>
         protected void EnsureStorage()
         {
-            if (runstackpos < runtrackcount * 4)
+            int tmpmultiplicationsize = runtrackcount * 4;
+            if (runstackpos < tmpmultiplicationsize)
                 DoubleStack();
-            if (runtrackpos < runtrackcount * 4)
+            if (runtrackpos < tmpmultiplicationsize)
```

Comment:
I don't think this is a worthwhile change.  It's not going to improve the generated asm, and it makes the code harder to read.  Let's let the compiler do its thing.

Reply:
Okey, you're right.
I just wanted to help improve the Regex class.

Reply:
And we appreciate that.  Thanks for your attempt.